### DISCUSSION TITLE
ScriptReference/EditorGUIUtility-systemCopyBuffer

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#238] [Notes]
# ScriptReference/EditorGUIUtility-systemCopyBuffer

https://docs.unity3d.com/ScriptReference/EditorGUIUtility-systemCopyBuffer.html

**Comment by mvi** [comment]
Note that as of Unity 5.2 this is just a wrapper for [`GUIUtility.systemCopyBuffer`](https://docs.unity3d.com/ScriptReference/GUIUtility-systemCopyBuffer.html).
